frictionless/plugins/ckan.py

```diff
@@ -230,32 +223,61 @@ def write_table_convert_field_type(self, type):
             "object": "json",
             "array": "text[]",
             "geojson": "json",
-            "date": "date",
+            "date": "text",
```

Comment:
I made this `"date": "text",` instead of `"date": "date",` because I was hitting
`The data was invalid: invalid input syntax for type date: "2015"`
on "date_year" in the types fixture if I used `date` (although "date" from the fixture does work if I set `"date": "date",` here).
I'm not sure what the right thing to do here is?
- What I've done here (`"date": "text",`)
- Special-case `date` somewhere if the `format` param is anything other than `Y-m-d` format
- Is this actually a bug in the date type/write_cell ?

I don't think I understand well enough what you're trying to achieve clearly enough to make a judgement on it.